crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -28,12 +29,14 @@ const INDENT: &str = "\t";
 pub fn cli() -> clap::Command {
     clap::Command::new("generate")
         .about("Generate client files for a spacetime module.")
+        .override_usage("spacetime generate --lang <LANG> --out-dir <DIR> [--project-path <DIR> | --wasm-file <PATH>]")
```

Comment:
We're breaking the API here a bit - some scripts in the BitCraft project assume that you don't have to pass a path and that it will use the current directory as the default. 

IMO I think we should maintain the old behavior of defaulting to the current directory.

Reply:
We would also need to update all of the quickstarts, tutorials, docs, etc. on the website after this merges otherwise people will run into the same issue that I ran into:
```
Generating C# files
error: the following required arguments were not provided:
  <--wasm-file <wasm_file>|--project-path <project_path>|--json-module [<json_module>]>

Usage: spacetime generate --lang <LANG> --out-dir <DIR> [--project-path <DIR> | --wasm-file <PATH>]
```

Reply:
If you'd like to push a fix to this branch, feel free. I have no strong feelings about this; the only required change is that we need the new `--json-module` as an alternative to `--project-path`.

Reply:
> If you'd like to push a fix to this branch, feel free. I have no strong feelings about this; the only required change is that we need the new `--json-module` as an alternative to `--project-path`.

ok understood, I will push the fix